app/src/main/java/ch/epfl/sweng/favors/main/LoggedInScreen.java

```diff
@@ -112,6 +115,8 @@ public void onPropertyChanged(Observable sender, int propertyId) {
             binding.navView.setCheckedItem(R.id.favors);
         }
 
+        //binding.toolbar.setBackgroundColor(LocalPreferences.getInstance().getColor());
```

Comment:
Do we still need this ligne? What does it do?

Reply:
oh that rest of garbage

Reply:
thanks